deploy-agent/deployd/common/stats.py

```diff
@@ -31,9 +32,72 @@ def create_stats_timer(stats, sample_rate, tags):
         return DefaultStatsdTimer()
 
 
+def create_sc_timing(stat, value, sample_rate, tags):
+    if IS_PINTEREST:
+        from pinstatsd.statsd import sc
+        return sc.timing(stat, value, sample_rate=sample_rate, tags=tags)
+    else:
+        return
+
+
 def create_sc_increment(stats, sample_rate, tags):
     if IS_PINTEREST:
         from pinstatsd.statsd import sc
         sc.increment(stats, sample_rate, tags)
     else:
         return
+
+
+def create_sc_gauge(stat, value, sample_rate, tags):
+    if IS_PINTEREST:
+        from pinstatsd.statsd import sc
+        sc.gauge(stat, value, sample_rate=sample_rate, tags=tags)
+    else:
+        return
+
+
+class TimeElapsed():
+    """ keep track of elapsed time in seconds """
+
+    def __init__(self):
+        self._time_start = self._timer()
+        self._time_now = None
+        self._time_elapsed = float(0)
+        self._time_pause = None
+
+    def get(self):
```

Comment:
just confirming this returns a float for the time in seconds?  Given historically statsd etc has only worked on ints it may be prudent to convert this to an int

Reply:
tested both, decided on casting to stay consistent.  Thank you for the recommendation.